gltfTutorial/gltfTutorial_020_Skins.md

```diff
@@ -161,11 +163,11 @@ The `"JOINTS_0"` attribute refers to an accessor that contains the indices of th
     Vertex 8:  0, 1, 0, 0,
     Vertex 9:  0, 1, 0, 0,
 
-This means that every vertex should be influenced by joint 0 and joint 1, except for the two vertices at the bottom. (The last 2 components of each vector are ignored here. If there were multiple joints, then one entry of this accessor could, for example, contain
+This means that every vertex may be influenced by joint 0 and joint 1  (except for the two vertices at the bottom, which may only be influenced by joint 0). The last 2 components of each vector are ignored here. If there were multiple joints, then one entry of this accessor could, for example, contain

```

Comment:
```suggestion
This means that every vertex may be influenced by joint 0 and joint 1, except the first two vertices are influenced only by joint 0, and the last two vertices are influenced only by joint 1. The last 2 components of each vector are ignored here. If there were multiple joints, then one entry of this accessor could, for example, contain
```

Reply:
Just a suggestion for how to refer to the unique vertices.

Reply:
I struggled to find the best wording here, because someone might read these numbers, and wonder _what they mean_. And from the joint numbers alone, it is not clear why the last vertices should *only* be influenced by joint 1. That only becomes clear when taking the _weights_ into account.